lib/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ *******************************************************
+ *                                                     *
+ *   Copyright (C) Microsoft. All rights reserved.     *
+ *                                                     *
+ *******************************************************
+ */
+
+var LiveReload = require('./livereload');
+
+/**
+ * @constructor
+ * @param {string} projectRoot - root folder of the project
+ * @param {string} platformWWW - www directories of the platforms
+ * @param {Object} options - Options to initialize LiveReload with - These are to be passed to the underlying BrowserSync.
+ *                           See http://www.browsersync.io/docs/options/ for documentation
+ */
+module.exports = function(projectRoot, platformWWWs, options) {
+    return new LiveReload(projectRoot, platformWWWs, options);
```

Comment:
Instead of returning an instance - it's common pattern for node modules to return a create method instead. Here are some good ideas on design patterns for node: http://bites.goodeggs.com/posts/export-this/#monkey_patch